webanno-brat/src/main/java/de/tudarmstadt/ukp/clarin/webanno/brat/render/model/Entity.java

```diff
@@ -52,32 +52,33 @@ public Entity()
     }
 
     public Entity(int aId, String aType, Offsets aOffsets, String aLabelText, String aColor,
-            String aHoverSpantext)
+            String aHoverSpantext, Boolean aActionButtons)
```

Comment:
Do we really need to have the actionButtons flag in *all* constructors? 
Also, I'd make it a `boolean` instead of a `Boolean` here and later internally treat `false` as `null` in the attributes object.